doc/configuration.md

```diff
@@ -2,9 +2,11 @@
 
 VS Code settings to configure Joyride.
 
-There is no configuration. 😀
+* `joyride.nReplHostAddress`, the host address to bind the nRepl server to
```

Comment:
I would call this option:

```
joyride.nreplHostAddress`
```

since in Clojure code we write `nrepl`, not `n-repl`